swarm/CHANGELOG.md

```diff
@@ -9,19 +9,21 @@
 - Add `estblished_in` to `SwarmEvent::ConnectionEstablished`. See [PR 3134].
 
 - Remove deprecated `inject_*` methods from `NetworkBehaviour` and `ConnectionHandler`.
-  see [PR 3264].
+  Make the implementation of `on_swarm_event` and `on_connection_handler_event`
+  both mandatory. See [PR 3264] and [PR 3364].
```

Comment:
```suggestion
  Make the implementation of `on_connection_handler_event`
  mandatory and thus consistent with `on_swarm_event`. See [PR 3264] and [PR 3364].
```

I find the wording hard to understand, i.e. I would expect this change to touch both methods, whereas in reality it doesn't.

Reply:
Ah, right, my bad. Thanks for clarifying.